Services/CO.CDP.EntityVerification/Api/PponEndpointExtensions.cs

```diff
@@ -34,7 +34,31 @@ await useCase.Execute(new LookupIdentifierQuery(identifier))
                 operation.Responses["404"].Description = "Identifier not found.";
                 operation.Responses["500"].Description = "Internal server error.";
                 return operation;
-            });
+            });       
+    }
+
+    public static void UseRegistriesEndpoints(this WebApplication app)
+    {
+        app.MapGet("/registries/{countrycode}",
+           [Authorize(Policy = "OneLoginPolicy")]
+        async (string countrycode, IUseCase<string, IEnumerable<IdentifierRegistries>> useCase) =>
+           await useCase.Execute(countrycode)
+                  .AndThen(identifiers => identifiers != null ? Results.Ok(identifiers) : Results.NotFound()))
```

Comment:
The return type of the use case is IEnumerable<Model.IdentifierRegistries>, so it can never be `null`.

Reply:
Done